.github/workflows/ci.yml

```diff
@@ -0,0 +1,132 @@
+name: Alfresco Transform Core CI
+
+on:
+  push:
+    branches:
+      - feature/**
+      - fix/**
+      - SP/**
```

Comment:
Wouldn't it be good to remove `SP/**` and `HR/**` from push branches since they are already in master_release.yml ?